ds_judgements_public_ui/templates/layout_judgment_html.html

```diff
@@ -24,13 +24,15 @@
     {% include 'includes/gtm/gtm_head.html' %}
 </head>
 <body>
+{% include 'includes/cookie_consent/cookie_banner.html' %}
 <a id="skip-to-main-content" href="#main-content">{% translate "skiplink" %}</a>
 {% include 'includes/gtm/gtm_body.html' %}
 {% include 'includes/phase_banner.html' %}
 <header class="page-header">
   {% include 'includes/breadcrumbs.html' with current=context.page_title title=context.page_title link=request.path %}
   {% include 'includes/logo.html' %}
 </header>
+<a id="skip-to-main-content" href="#main-content">Skip to Main Content</a>
```

Comment:
Should this be here? Looks like a repeat of line 28?

Reply:
Thanks @Floppy I've removed the duplicate now 👍🏽